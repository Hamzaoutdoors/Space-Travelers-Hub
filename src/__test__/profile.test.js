import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyRocket from '../components/profile/MyRockets';
import store from '../redux/configureStore';

test('rockets list renders properly', () => {
  const tree = renderer
    .create(<Provider store={store}><MyRocket /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
