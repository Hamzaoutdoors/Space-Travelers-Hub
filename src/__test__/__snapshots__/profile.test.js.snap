// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rockets list renders properly 1`] = `
<div
  className="my-rockets"
>
  <h2
    className="mb-4"
  >
    My Rockets
  </h2>
  <div
    className="table-responsive"
  >
    <table
      className="my-items table table-sm table-bordered table-hover"
    >
      <tbody />
    </table>
  </div>
</div>
`;
